pythran/tests/test_scipy_func.py

```diff
@@ -0,0 +1,51 @@
+from test_env import TestEnv
+import numpy
+from pythran.typing import NDArray, List
+
+
+@TestEnv.module
+class TestScipyFunc(TestEnv):
+    def test_special_gamma0(self):
+        self.run_test(
+            """
+            def scipy_special_gamma0(a):
+                import scipy.special
+                return scipy.special.gamma(a)
+            """,
+            3.5,
+            scipy_special_gamma0=[float]
+        )
+
+    def test_special_gamma1(self):
+        self.run_test(
+            """
+            def scipy_special_gamma1(a):
+                import scipy.special
+                return scipy.special.gamma(a)
+            """,
+            numpy.array([1.1, 2.2, 3.3]),
+            scipy_special_gamma1=[NDArray[float, :]]
+        )
+
+    def test_special_gammaln0(self):
+        self.run_test(
+            """
+            def scipy_special_gammaln0(a):
+                import scipy.special
+                return scipy.special.gammaln(a)
+            """,
+            3.5,
+            scipy_special_gammaln0=[float]
+        )
+
+    def test_special_gammaln1(self):
+        self.run_test(
+            """
+            def scipy_special_gammaln1(a):
+                import scipy.special
+                return scipy.special.gammaln(a)
+            """,
+            numpy.array([1.1, 2.2, 3.3]),
+            scipy_special_gammaln1=[NDArray[float, :]]
+        )
+
```

Comment:
Maybe it's better to use the existing test env in ``pythran/tests/test_numpy_ufunc_unary.py`` instead? This requires some slight refactoring but still it's more generic.